vm/package/deploy.rs

```diff
@@ -67,7 +56,7 @@ impl<N: Network> DeployRequest<N> {
 impl<N: Network> Serialize for DeployRequest<N> {
     /// Serializes the deploy request into string or bytes.
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
-        let mut request = serializer.serialize_struct("DeployRequest", 1)?;
```

Comment:
this looks suspicious why do we need to manually keep the amount of fields in sync?

Reply:
https://serde.rs/impl-serialize.html#serializing-a-struct

It's a size hint that some libraries implementing `Serialize` will use to determine the map allocation